Directory.Build.props

```diff
@@ -28,7 +27,7 @@
   </PropertyGroup>
   <PropertyGroup Label="Condition Variables" >
     <IsNetStandard>$(NetStandardVersions.Contains('$(TargetFramework)'))</IsNetStandard>
-    <IsNetCore>$(NetCoreVersions.Contains('$(TargetFramework)'))</IsNetCore>
+    <IsNetCore>$(DefaultNetCoreVersion.Contains('$(TargetFramework)'))</IsNetCore>
```

Comment:
I'm not sure this will work as expected, needs verification.

Reply:
re-added NetCoreVersions to align with previous behaviour